docs/Course Wiki/ECE Course Offerings/ECE340.md

```diff
@@ -1,35 +1,46 @@
 # ECE340
+##Semiconductor Electronics.
+
+ECE 340 is the introductory semiconductors course that is required for EE and a 1-of-6 for CE.
```

Comment:
ECE340 (COURSE NAME HERE) is a 3-credit-hour course that is specifically required for EEs as part of the Electrical Engineering Core and satisfies the 1-of-6 Electrical Engineering Foundations Course requirement for CEs. It is offered in the fall, spring, and summer.